apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.html.ts

```diff
@@ -84,6 +86,27 @@ export function InstructorQuestionPreview({
         : ''}
       <div class="row">
         <div class="col-lg-9 col-sm-12">
+          ${readmeHtml
+            ? html`
+                <div class="card mb-3">
+                  <div class="card-header bg-secondary text-white d-flex align-items-center">
+                    <button
+                      class="btn btn-secondary btn-xs show-hide-btn"
+                      data-bs-toggle="collapse"
+                      data-bs-target="#readme-card-body"
+                      aria-expanded="true"
+                      aria-controls="#readme-card-body"
+                    >
+                      <i class="fa fa-angle-up fa-fw expand-icon"></i>
+                    </button>
+                    <h2>README</h2>
+                  </div>
+                  <div class="card-body collapse show" id="readme-card-body">
+                    <div class="markdown-body">${unsafeHtml(readmeHtml)}</div>
```

Comment:
IMO you'd benefit from styles similar to this here:

https://github.com/PrairieLearn/PrairieLearn/blob/09b9386f2e32621310fb2f730435d555dec39870/apps/prairielearn/src/pages/studentAssessment/studentAssessment.html.ts#L31-L33